### PR TITLE
lowercase addresses too

### PR DESCRIPTION
**Pull request by fgregg**
Wait for the new version of usaddress that cathy is pushing.

### CODE REVIEW

File: api/utils/helpers.py [hunk 1]
```diff
@@ -329,7 +329,7 @@ def getMatches(session_id, records):
     if match_ids :
         entities = ''' 
             SELECT 
-              e.entity_id
+              e.entity_id,
```
Comment (fgregg) [inline api/utils/helpers.py]: Fixes bug introduced in de121f0f1ec70352d683f60cf63f518cf842515e

**Comment by fgregg** [comment]
Much less depressing final review count estimator.

Now the estimate of stuff left to review is the 

estimate of how much of the completely unseen stuff will need human review **plus** the size of the current queue count.

Small improvement in accuracy, big payoff here is that when you review a record, the estimate actually decrements by one since the size of the current queue count goes down.

**Comment by fgregg** [comment]
I think this is ready to deploy @evz

**Comment by fgregg** [comment]
please update usaddress on the server when you deploy this.

**Comment by evz** [comment]
OK, this is deployed

**Pull request merged**